src/Domain.Linnapps/Models/ProductionTriggerLevel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Linn.Products.Domain.Linnapps.Models
```

Comment:
There's another `ProductionTriggerLevel` class in `Domain.Linnapps`  (not the Models folder) - could this cause issues?

Reply:
Is there? Are you thinking of the one in the production solution?

Reply:
Ahh yes...